test/functional/wallet_labels.py

```diff
@@ -29,8 +29,8 @@ def run_test(self):
 
         # Note each time we call generate, all generated coins go into
         # the same address, so we call twice to get two addresses w/50 each
-        node.generate(1)
-        node.generate(101)
+        node.generatetoaddress(nblocks=1, address=node.getnewaddress(label='coinbase'))
+        node.generatetoaddress(nblocks=101, address=node.getnewaddress(label='coinbase'))
```

Comment:
maybe put the `coinbase_address` as a variable?

Reply:
The intent here is to have two addresses with 50 bitcoin each (see comment immediately above).

Reply:
makes sense.